config/default/manager_auth_proxy_patch.yaml

```diff
@@ -0,0 +1,26 @@
+# This patch inject a sidecar container which is a HTTP proxy for the
+# controller manager, it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
```

Comment:
I didn't use this in the konk-operator chart. Maybe it would help with rbac issues for the charts deployed by the operator.